README.md

```diff
@@ -2,3 +2,6 @@ marble-o-man
 ============
 
 Messing around with very basic algorithm to try to do marbling -- kind of abandoned this approach as it was too ad-hoc and did not allow for much in the way of chaotic complexity
+
+Expected output:
+![output](https://github.com/user-attachments/assets/c2d715f7-cc3a-46b4-a3b2-9ab7ca1bca0f)
```

Comment:
I don't really like this linked to a github user attachment, as if you clone the repo and try to view it offline, it won't be accessible.